tests/tools/onert_train/README.md

```diff
@@ -18,6 +18,79 @@ sudo apt install -y libhdf5-dev libboost-program-options-dev
 
 ## Usage
 
-### Simple train
+`onert_train` takes a model(*.circle or nnpackage), training data files and training parameters.  
 
-### Simple inference to trained model
+You could train your model using the command like below.  
+```bash
+$ onert_train \
+--path [circle file or nnpackage] \
+--load_input:raw [training input data] \
+--load_expected:raw [training output data] \
+--batch_size 32 \ 
+--epoch 5 \
+--optimizer 1 \             # sgd
+--learning_rate 0.01 \   
+--loss 2 \                  # cateogrical crossentropy
+--loss_reduction_type 1     # sum over batch size
+```
+`onert_train --help` would help you to set each parameter.
+
```

Comment:
I tried to write a readme for ONE developers, but who are new to the training feature.

I thought simple usage wasn't enough - they might feel confused about preparing training data.
So, I added an Example part below.